src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,22 +36,84 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            // Poll for model results and set the results in the scenario when done.
+            var pollModel = function(scenario) {
```

Comment:
I can move pollModel into scenarioModel, as long as I can give scenarioModel a reference to project. I'll look into how to do that. I have some ideas for testing. Can you come over some time and we can discuss them?